application/performance/locust/coldstartscenario.py

```diff
@@ -0,0 +1,68 @@
+import json
+from locust import task, FastHttpUser, constant
+import config as config
+
+
+class FiveDest(FastHttpUser):
+    weight = 80
+    wait_time = constant(300)
```

Comment:
Is this wait_time mean that there is always 300s between requests?
The scenario asks for an increasing wait between each request, should it increase by 60 secs between each request to understand the point at which the cold start kicks in?